language/de/ucp.php

```diff
@@ -427,6 +427,7 @@
 	'NO_BOOKMARKS_SELECTED'		=> 'Du hast keine Lesezeichen ausgewählt.',
 	'NO_EDIT_READ_MESSAGE'		=> 'Diese Private Nachricht kann nicht geändert werden, da sie bereits gelesen wurde.',
 	'NO_EMAIL_USER'				=> 'Es existiert kein Benutzer mit dieser Kombination aus Benutzernamen und E-Mail-Adresse.',
+	'EMAIL_NOT_UNIQUE'			=> 'Die angebene E-Mail-Adresse wird von mehreren Benutzern verwendet. Du musst zusätzlich den Benutzernamen angeben.',
```

Comment:
```suggestion
	'EMAIL_NOT_UNIQUE'			=> 'Die angegebene E-Mail-Adresse ist mehreren Benutzern zugeordnet. Du musst zusätzlich den Benutzernamen angeben.',
```

Ist zwar nicht wörtlich - aber imho korrekter :-)